sdk/contentsafety/azure-ai-contentsafety/src/main/java/com/azure/ai/contentsafety/ContentSafetyClient.java

```diff
@@ -440,221 +163,102 @@ public AnalyzeTextResult analyzeText(AnalyzeTextOptions body) {
-     * @return the response of adding blockItems to text blocklist.
+     * @return the image analysis response.
      */
     @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public AddBlockItemsResult addBlockItems(String blocklistName, AddBlockItemsOptions addBlockItemsOptions) {
-        // Generated convenience method for addBlockItemsWithResponse
+    public AnalyzeImageResult analyzeImage(AnalyzeImageOptions body) {
+        // Generated convenience method for analyzeImageWithResponse
         RequestOptions requestOptions = new RequestOptions();
-        return addBlockItemsWithResponse(blocklistName, BinaryData.fromObject(addBlockItemsOptions), requestOptions)
+        return analyzeImageWithResponse(BinaryData.fromObject(body), requestOptions)
                 .getValue()
-                .toObject(AddBlockItemsResult.class);
+                .toObject(AnalyzeImageResult.class);
     }
 
     /**
-     * Remove BlockItems From Text Blocklist
+     * Analyze Image
      *
-     * <p>Remove blockItems from a text blocklist. You can remove at most 100 BlockItems in one request.
+     * <p>A synchronous API for the analysis of potentially harmful image content. Currently, it supports four
+     * categories: Hate, SelfHarm, Sexual, and Violence.
      *
-     * @param blocklistName Text blocklist name.
-     * @param removeBlockItemsOptions The request of removing blockItems from text blocklist.
+     * @param blobUri A string of blob url.
      * @throws IllegalArgumentException thrown if parameters fail the validation.
      * @throws HttpResponseException thrown if the request is rejected by server.
      * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
      * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the image analysis response.
      */
-    @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public void removeBlockItems(String blocklistName, RemoveBlockItemsOptions removeBlockItemsOptions) {
-        // Generated convenience method for removeBlockItemsWithResponse
+    public AnalyzeImageResult analyzeImage(String blobUri) {
+        // Customized convenience method for analyzeImageWithResponse
         RequestOptions requestOptions = new RequestOptions();
-        removeBlockItemsWithResponse(blocklistName, BinaryData.fromObject(removeBlockItemsOptions), requestOptions)
-                .getValue();
+        AnalyzeImageOptions body = new AnalyzeImageOptions(new ContentSafetyImageData().setBlobUri(blobUri));
+        return analyzeImageWithResponse(BinaryData.fromObject(body), requestOptions)
+            .getValue()
+            .toObject(AnalyzeImageResult.class);
```

Comment:
nit, it usually better to call `analyzeImage`

Reply:
updated